documentation/docs/14-configuration.md

```diff
@@ -132,8 +129,8 @@ Options related to [creating a package](#packaging).
 
 - `dir` - output directory
 - `emitTypes` - by default, `svelte-kit package` will automatically generate types for your package in the form of `d.ts.` files. While generating types is configurable, we believe it is best for the ecosystem quality to generate types, always. Please make sure you have a good reason when setting it to `false` (for example when you want to provide handwritten type definitions instead).
-- `exports` - contains a `includes` and a `excludes` array which specifies which files to mark as exported from the `exports` field of the `package.json`
-- `files` - contains a `includes` and a `excludes` array which specifies which files to process and copy over when packaging
+- `exports` - specifies which files to mark as exported from the `exports` field of the `package.json`, can be set to `null` and will use existing `exports` field if available, or an object that contains an `includes` and an `excludes` array and will merge existing `exports`, with the value from `package.json` taking precedence
+- `files` - contains an `includes` and an `excludes` array which specifies which files to process and copy over when packaging
```

Comment:
Thinking of expanding this section to use `####` headings instead

Reply:
I think it's not necessary yet. But the sentence could be split into multiple sentences to make reading a little easier.